<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ImportDataMaster</class>
 <widget class="QWizard" name="ImportDataMaster">
  <property name="windowModality">
   <enum>Qt::WindowModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>655</width>
    <height>469</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>480</width>
    <height>365</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Импорт данных</string>
  </property>
  <widget class="QWizardPage" name="wizardPage1">
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QLabel" name="label">
        <property name="font">
         <font>
          <pointsize>12</pointsize>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string>Импорт обучающей выборки</string>
        </property>
        <property name="textFormat">
         <enum>Qt::AutoText</enum>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <spacer name="verticalSpacer_5">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeType">
       <enum>QSizePolicy::Fixed</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>20</width>
        <height>10</height>
       </size>
      </property>
     </spacer>
    </item>
    <item>
     <widget class="QLabel" name="label_10">
      <property name="text">
       <string>Название набора данных*</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="nameData">
      <property name="text">
       <string>Набор данных</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_2">
      <property name="text">
       <string>Выберите файл с обучающей выборкой*</string>
      </property>
     </widget>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QLineEdit" name="tbFilePath">
        <property name="enabled">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeType">
         <enum>QSizePolicy::Fixed</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>13</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="btOpenDataFile">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>30</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>30</height>
         </size>
        </property>
        <property name="text">
         <string>Обзор</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <spacer name="verticalSpacer">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeType">
       <enum>QSizePolicy::Fixed</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>20</width>
        <height>10</height>
       </size>
      </property>
     </spacer>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QLabel" name="label_9">
        <property name="text">
         <string>Лист</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QSpinBox" name="spinLoadPage">
        <property name="minimum">
         <number>1</number>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer_5">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeType">
         <enum>QSizePolicy::Fixed</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>10</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>Импорт со строки</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QSpinBox" name="spinRowStart">
        <property name="minimum">
         <number>1</number>
        </property>
        <property name="maximum">
         <number>16777215</number>
        </property>
        <property name="value">
         <number>1</number>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer_6">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeType">
         <enum>QSizePolicy::Fixed</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>5</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="Line" name="line">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer_4">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeType">
         <enum>QSizePolicy::Fixed</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>5</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QCheckBox" name="chbFirstRowIsHeader">
        <property name="text">
         <string>Первая строка - заголовок</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer_2">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </item>
    <item>
     <spacer name="verticalSpacer_2">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeType">
       <enum>QSizePolicy::Fixed</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>20</width>
        <height>15</height>
       </size>
      </property>
     </spacer>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <item>
       <widget class="QLabel" name="label_4">
        <property name="text">
         <string>Предпросмотр обучающей выборки</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer_3">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QTableView" name="preViewTable">
      <property name="layoutDirection">
       <enum>Qt::LeftToRight</enum>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWizardPage" name="wizardPage2">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <item row="2" column="0">
     <layout class="QHBoxLayout" name="horizontalLayout_4">
      <item>
       <widget class="QListWidget" name="columnsWidget"/>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_3">
        <property name="sizeConstraint">
         <enum>QLayout::SetMinimumSize</enum>
        </property>
        <property name="bottomMargin">
         <number>0</number>
        </property>
        <item>
         <widget class="QLabel" name="label_5">
          <property name="text">
           <string>Имя столбца</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="tbColumnName"/>
        </item>
        <item>
         <widget class="QLabel" name="label_6">
          <property name="text">
           <string>Тип данных</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="cb_data_type">
          <item>
           <property name="text">
            <string>[0/1] Логический</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>[12.5] Числовой</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>[abc] Строковый </string>
           </property>
          </item>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_7">
          <property name="text">
           <string>Назначение</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="cb_input_param">
          <item>
           <property name="text">
            <string>Информационный</string>
           </property>
           <property name="icon">
            <iconset resource="../icons.qrc">
             <normaloff>:/icons/icons/info.png</normaloff>:/icons/icons/info.png</iconset>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Входной</string>
           </property>
           <property name="icon">
            <iconset resource="../icons.qrc">
             <normaloff>:/icons/icons/input.png</normaloff>:/icons/icons/input.png</iconset>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Выходной</string>
           </property>
           <property name="icon">
            <iconset resource="../icons.qrc">
             <normaloff>:/icons/icons/output.png</normaloff>:/icons/icons/output.png</iconset>
           </property>
          </item>
         </widget>
        </item>
        <item>
         <spacer name="verticalSpacer_3">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>40</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
     </layout>
    </item>
    <item row="0" column="0">
     <widget class="QLabel" name="label_8">
      <property name="font">
       <font>
        <pointsize>12</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Настройка параметров атрибутов</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <spacer name="verticalSpacer_4">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeType">
       <enum>QSizePolicy::Fixed</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>20</width>
        <height>10</height>
       </size>
      </property>
     </spacer>
    </item>
   </layout>
  </widget>
  <widget class="QWizardPage" name="wizardPage">
   <layout class="QVBoxLayout" name="verticalLayout_5">
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_4">
      <item>
       <widget class="QLabel" name="label_11">
        <property name="font">
         <font>
          <pointsize>12</pointsize>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string>Импорт тестовой выборки</string>
        </property>
        <property name="textFormat">
         <enum>Qt::AutoText</enum>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="chbEnableTest">
        <property name="text">
         <string>Использовать обучающую выборку</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="verticalSpacer_6">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeType">
         <enum>QSizePolicy::Fixed</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>5</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QWidget" name="testWidget" native="true">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_7">
         <item>
          <widget class="QLabel" name="label_14">
           <property name="text">
            <string>Выберите файл с тестовой выборкой*</string>
           </property>
          </widget>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_7">
           <item>
            <widget class="QLineEdit" name="tbFileTestDataPath">
             <property name="enabled">
              <bool>false</bool>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="btOpenDataTestFile">
             <property name="minimumSize">
              <size>
               <width>0</width>
               <height>30</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>16777215</width>
               <height>30</height>
              </size>
             </property>
             <property name="text">
              <string>Обзор</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <spacer name="verticalSpacer_7">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeType">
            <enum>QSizePolicy::Fixed</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>20</width>
             <height>10</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_6">
           <item>
            <widget class="QLabel" name="label_12">
             <property name="text">
              <string>Лист</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QSpinBox" name="spin_test_load_page">
             <property name="minimum">
              <number>1</number>
             </property>
            </widget>
           </item>
           <item>
            <spacer name="horizontalSpacer_7">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeType">
              <enum>QSizePolicy::Fixed</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>10</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
           <item>
            <widget class="QLabel" name="label_13">
             <property name="text">
              <string>Импорт со строки</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QSpinBox" name="spin_test_row_start">
             <property name="minimum">
              <number>1</number>
             </property>
             <property name="maximum">
              <number>16777215</number>
             </property>
            </widget>
           </item>
           <item>
            <spacer name="horizontalSpacer_8">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeType">
              <enum>QSizePolicy::Fixed</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>10</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
           <item>
            <widget class="QCheckBox" name="chb_test_first_row_is_header">
             <property name="text">
              <string>Первая строка - заголовок</string>
             </property>
            </widget>
           </item>
           <item>
            <spacer name="horizontalSpacer_10">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>40</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
          </layout>
         </item>
         <item>
          <spacer name="verticalSpacer_8">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeType">
            <enum>QSizePolicy::Fixed</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>20</width>
             <height>15</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QLabel" name="label_15">
           <property name="text">
            <string>Предпросмотр тестовой выборки</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QTableView" name="preViewTestTable">
           <property name="layoutDirection">
            <enum>Qt::LeftToRight</enum>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QWizardPage" name="wizardPage_2">
   <layout class="QVBoxLayout" name="verticalLayout_6">
    <item>
     <widget class="QLabel" name="label_16">
      <property name="font">
       <font>
        <pointsize>12</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Настройка атрибутов тестовой выборки</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QTableWidget" name="testForLearnColumns"/>
    </item>
   </layout>
  </widget>
 </widget>
 <resources>
  <include location="../icons.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>btOpenDataFile</sender>
   <signal>clicked()</signal>
   <receiver>ImportDataMaster</receiver>
   <slot>open_data_file()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>572</x>
     <y>99</y>
    </hint>
    <hint type="destinationlabel">
     <x>316</x>
     <y>234</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>chbFirstRowIsHeader</sender>
   <signal>clicked()</signal>
   <receiver>ImportDataMaster</receiver>
   <slot>is_have_headers()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>475</x>
     <y>212</y>
    </hint>
    <hint type="destinationlabel">
     <x>316</x>
     <y>234</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>spinRowStart</sender>
   <signal>valueChanged(int)</signal>
   <receiver>ImportDataMaster</receiver>
   <slot>is_value_row_start_change()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>300</x>
     <y>213</y>
    </hint>
    <hint type="destinationlabel">
     <x>316</x>
     <y>234</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>spinLoadPage</sender>
   <signal>valueChanged(int)</signal>
   <receiver>ImportDataMaster</receiver>
   <slot>is_value_page_change()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>88</x>
     <y>213</y>
    </hint>
    <hint type="destinationlabel">
     <x>316</x>
     <y>234</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>columnsWidget</sender>
   <signal>itemSelectionChanged()</signal>
   <receiver>ImportDataMaster</receiver>
   <slot>is_list_columns_item_selected()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>166</x>
     <y>231</y>
    </hint>
    <hint type="destinationlabel">
     <x>316</x>
     <y>234</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>cb_input_param</sender>
   <signal>currentIndexChanged(int)</signal>
   <receiver>ImportDataMaster</receiver>
   <slot>is_input_param_change()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>467</x>
     <y>212</y>
    </hint>
    <hint type="destinationlabel">
     <x>316</x>
     <y>234</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>cb_data_type</sender>
   <signal>currentIndexChanged(int)</signal>
   <receiver>ImportDataMaster</receiver>
   <slot>is_data_type_change()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>467</x>
     <y>156</y>
    </hint>
    <hint type="destinationlabel">
     <x>316</x>
     <y>234</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>tbColumnName</sender>
   <signal>textChanged(QString)</signal>
   <receiver>ImportDataMaster</receiver>
   <slot>is_name_column_change()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>467</x>
     <y>97</y>
    </hint>
    <hint type="destinationlabel">
     <x>316</x>
     <y>234</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>tbColumnName</sender>
   <signal>editingFinished()</signal>
   <receiver>ImportDataMaster</receiver>
   <slot>is_name_column_finished()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>467</x>
     <y>97</y>
    </hint>
    <hint type="destinationlabel">
     <x>316</x>
     <y>234</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>chbEnableTest</sender>
   <signal>stateChanged(int)</signal>
   <receiver>ImportDataMaster</receiver>
   <slot>is_test_enable()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>354</x>
     <y>54</y>
    </hint>
    <hint type="destinationlabel">
     <x>354</x>
     <y>234</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btOpenDataTestFile</sender>
   <signal>clicked()</signal>
   <receiver>ImportDataMaster</receiver>
   <slot>open_test_data_file()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>597</x>
     <y>112</y>
    </hint>
    <hint type="destinationlabel">
     <x>327</x>
     <y>234</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>spin_test_load_page</sender>
   <signal>valueChanged(int)</signal>
   <receiver>ImportDataMaster</receiver>
   <slot>is_test_value_page_change()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>70</x>
     <y>161</y>
    </hint>
    <hint type="destinationlabel">
     <x>327</x>
     <y>234</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>spin_test_row_start</sender>
   <signal>valueChanged(int)</signal>
   <receiver>ImportDataMaster</receiver>
   <slot>is_test_value_row_start_change()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>243</x>
     <y>161</y>
    </hint>
    <hint type="destinationlabel">
     <x>327</x>
     <y>234</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>chb_test_first_row_is_header</sender>
   <signal>stateChanged(int)</signal>
   <receiver>ImportDataMaster</receiver>
   <slot>is_test_have_headers()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>379</x>
     <y>161</y>
    </hint>
    <hint type="destinationlabel">
     <x>327</x>
     <y>234</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>open_data_file()</slot>
  <slot>validInput()</slot>
  <slot>is_have_headers()</slot>
  <slot>is_value_row_start_change()</slot>
  <slot>is_value_page_change()</slot>
  <slot>is_list_columns_item_selected()</slot>
  <slot>is_data_type_change()</slot>
  <slot>is_input_param_change()</slot>
  <slot>is_name_column_change()</slot>
  <slot>is_name_column_finished()</slot>
  <slot>open_test_data_file()</slot>
  <slot>is_test_value_row_start_change()</slot>
  <slot>is_test_value_page_change()</slot>
  <slot>is_test_have_headers()</slot>
  <slot>is_test_enable()</slot>
 </slots>
</ui>
